### PR TITLE
Intl.Locale: Unify code examples across the various getter properties

### PR DESCRIPTION
**Pull request by mattco98**
> What was wrong/why is this fix needed? (quick summary only)

Most of these examples had either incorrect variable names or inconsistent formatting (spacing, newlines, etc). 

> Issue number (if there is an associated issue)

N/A

> Anything else that could help us review it

I've chosen to give all of these variable names the generic name of `locale` for consistency. The examples are all extremely short (~2 lines), so there isn't really anything being lost compared to a more descriptive name (like `frBuddhistStr`). If the specific variable names are still desired, I can update this PR to include that change.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/intl/locale/locale/index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ property of the configuration object to your desired hour cycle type, and then p
 into the constructor:
 
 ```js
-let us12hour = new Intl.Locale("en-US", {hourCycle: "h12"});
+let us12hour = new Intl.Locale("en-US", { hourCycle: "h12" });
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/intl/locale/locale/index.md:58]: Is there any reason to keep `us12hour` here?

**Review by wbamberg — CHANGES_REQUESTED** [review]
This is a nice change, thank you!

There's one example where you haven't updated the variable name, I assume that's not intentional?

Thanks for fixing the variable name errors and the formatting inconsistencies. At first I didn't like the way you have changed the variable names to be generic, but having thought a bit I prefer it, so 👍 to that.

File: files/en-us/web/javascript/reference/global_objects/intl/locale/locale/index.md (hunk 1 above)
Comment (mattco98) [reply files/en-us/web/javascript/reference/global_objects/intl/locale/locale/index.md:58]: Ahh yeah that's not intentional, will fix

**Review by wbamberg — APPROVED** [review]
Thanks!

**Pull request merged**